src/infrastructure/database/modules/sqlite/repositories/sqlite-badge-class.repository.ts

```
@@ -5,242 +5,195 @@
     }
+
+    const partialExistingData = existingBadgeClass.toPartial();
+    const mergedData: Partial<BadgeClass> = {
+      ...partialExistingData,
+      ...badgeClass,
+    };
+
+    const mergedBadgeClass = BadgeClass.create(mergedData);
+
+    const record = { ...this.mapper.toPersistence(mergedBadgeClass) };
+    delete (record as Partial<typeof record>).id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)